source/hardware/controllers/gemini_g4v.rst

```diff
@@ -0,0 +1,250 @@
+Gemini G4V
+=====================
+
+.. figure:: ../../_static/controllers/gemini_g4v_topview.jpg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Behringer CMD Micro
+   :figclass: pretty-figures
+
+The Gemini G4V is a 2 deck controller that supports 4 virtual decks, a 4 way mixer and has a built-in 4 channel USB soundcard.
+It includes a Microphone input line, an Aux input line and an Booth output line. This lines are not going through the soundcard but mixed directly at the controller.
+They are not accessible nor controlled by Mixx so they will not appear in any recording nor is possible to control them via the Mixxx screen interface.
+
+
+-  `Manufacturer’s product page <http://geminisound.com/product/g4v>`__
+-  `Forum thread <https://www.mixxx.org/forums/viewtopic.php?f=6&t=12919>`__
+-  `Manufacturer’s User Guide <https://www.manualslib.com/manual/826563/Gemini-G4v.html>`__
+
+.. versionadded:: 2.3
+
+Mixxx sound hardware preferences
+--------------------------------
+
+========== =============
+Output     Channel
+========== =============
+Master     Channel 1 - 2
+Headphones Channel 3 - 4
+========== =============
+
+Mapping Description
+-------------------
+
+Most of the board's controls work as described in the G4V's manual, with a few exceptions. Snapshots of the board's labeling from the Manual are included below:
+
+
+.. figure:: ../../_static/controllers/gemini_g4v_controls.png
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Behringer CMD Micro
+   :figclass: pretty-figures
+
+.. figure:: ../../_static/controllers/gemini_g4v_frontview.jpg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Behringer CMD Micro
+   :figclass: pretty-figures
+
+.. figure:: ../../_static/controllers/gemini_g4v_controllist.jpg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Behringer CMD Micro
+   :figclass: pretty-figures
+
+Shift
+-----
+
+On each of the physical decks is a :hwlabel:`SHIFT` button.
+
+This button, when held, changes the behavior of various buttons on that deck, labeled under the respective button.
+
+Mixer
+-----
+
+In the middle of board, there are :hwlabel:`HIGH`, :hwlabel:`MID`, and :hwlabel:`LOW` equalizer knobs, :hwlabel:`GAIN`, and :hwlabel:`FILTER` knobs for the individual virtual decks that work as normal.
+
+The :hwlabel:`MASTER` and :hwlabel:`CUE MIX` knobs works as usual.
+
+The :hwlabel:`BOOTH` knob controls the gain of the booth audio output. This output is managed internally by the controller.
+
+Crossfader
+----------
+
+The :hwlabel:`CROSSFADER` works as usual.
```

Comment:
```suggestion
The Crossfader works as usual.
```

It's not labeled on the controller, right?

Reply:
You're correct, is not labelled in the controller, only in the documentation.

Reply:
Yes.